ckcp/hack/util/reset_ckcp.sh

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Pipeline Service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null || exit
+  pwd
+)"
+
+PLNSVC_DIR="$(dirname "$(dirname "$(dirname "$SCRIPT_DIR")")")"
+CKCP_DIR="$PLNSVC_DIR/ckcp"
+GITOPS_DIR="$PLNSVC_DIR/gitops"
+CONFIG="$PLNSVC_DIR/config/config.yaml"
+
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
+RESET_HARD="false"
+
+usage() {
+  printf "
+Usage:
+    %s [options]
+
+Scrap ckcp and free resources deployed by openshift_dev_setup.sh script.
+
+Mandatory arguments:
+    --kcp-kubeconfig KCP_KUBECONFIG
+        kubeconfig to the kcp-workspace.
+
+Optional arguments:
+    --reset-hard RESET_HARD
+        setting this option to 'true'
+        will aggresively remove operators deployed as well.
```

Comment:
will **aggressively** remove operators that were installed by the openshisft_dev_setup.sh script.